pkg/gateway/lb/handler/grpc/handler.go

```diff
@@ -475,6 +497,22 @@ func (s *server) SearchByID(ctx context.Context, req *payload.Search_IDRequest)
 	return res, nil
 }
 
+func (s *server) calculateNum(ctx context.Context, num uint32, ratio float32) (n uint32) {
+	min := float64(s.replica) / float64(len(s.gateway.Addrs(ctx)))
+	if ratio < 0.0 {
```

Comment:
It may this ? :thinking: 
```suggestion
	if ratio > 0.0 {
```

Reply:
why?

Reply:
ah, sorry, it's my misunderstanding. :sweat: